TerminalBoard.cpp

```diff
@@ -71,6 +71,7 @@ control(void *data)
 	BMessenger messenger(NULL, target);
 	while (true)
 	{
+		usleep(100);
 		int32 c = std::cin.get();
```

Comment:
This should block on the cin here and wait for keyboard input.
But, this is not the right way to handle keyboard input in a GUI application. Instead the KeyDown method in BView should be used.

Reply:
Wait, isn't this "TerminalBoard"? This looks like a terminal interface, the GUI one is in WindowBoard.

Reply:
In which case, the proper way would be to use select() on the stdin descriptor or something alike.